org.lflang/src/org/lflang/generator/GeneratorUtils.java

```diff
@@ -56,14 +56,10 @@ private GeneratorUtils() {
      * Return the target declaration found in the given resource.
      */
     public static TargetDecl findTarget(Resource resource) {
-        int cnt = 0;
         TargetDecl targetDecl = null;
         for (TargetDecl t : findAll(resource, TargetDecl.class)) { // getAllContents should never return null.
-            targetDecl = t;
-            cnt++;
+            return t;
```

Comment:
Wouldn't this also work?
```suggestion
            return findAll(resource, TargetDecl.class).iterator().next();
```

Reply:
this would also work.